### PR TITLE
Added n/a sections to docs

### PR DESCRIPTION
**Pull request by divyaamunugama**

### CODE REVIEW

File: en/docs/references/concepts/authentication/intro-ws-trust.md [hunk 1]
```diff
@@ -1,54 +1,70 @@
...
-issued token type is decided based on the type of token defined in the
-RST (Request Security Token). Currently, the Bearer Subject Confirmation
-and Holder-Of-Key subject confirmation methods, which are explained
-below, are supported. With Holder-Of-Key, both Symmetric and Asymmetric
-key types are supported. You can obtain tokens containing claims that
-hold certain information about the subject. These claims can be
-extracted from the profiles or through custom claim callbacks which can
-be registered at the Carbon runtime.
+
+<!-- With Holder-Of-Key, both Symmetric and Asymmetric key types are supported. You can obtain tokens containing claims that hold certain information about the subject. These claims can be extracted from the profiles or through custom claim callbacks which can be registered at the Carbon runtime. -->
```
Comment (divyaamunugama) [inline en/docs/references/concepts/authentication/intro-ws-trust.md:27]: The links in this section do not work.

File: en/docs/references/concepts/authentication/intro-ws-trust.md [hunk 2]
```diff
@@ -1,54 +1,70 @@
...
+
+This Security Token Service is capable of issuing SAML 1.1 and SAML 2.0
 tokens as recommended in WS-Trust and SAML Web Service Token Profile
-specifications.
+specifications. Both SAML 1.1 and SAML 2.0 token types are supported by default.
+
+The issued token type is decided based on the type of token defined in the
+RST (Request Security Token), and the token is issued through the RSTR (Request Security Token Response).
 
-<!-- The WSDL of this service can be accessed by clicking the URL having the
```
Comment (divyaamunugama) [inline en/docs/references/concepts/authentication/intro-ws-trust.md:12]: The links in this section do not work.

File: en/docs/references/concepts/authentication/intro-ws-trust.md [hunk 3]
```diff
@@ -1,54 +1,70 @@
...
-configure the generic STS to issue claim-based security tokens. A
-claim-based security token is a common way for applications to acquire
-and authenticate the identity information they need about users inside
-their organization, in other organizations, and on the Internet. This
-Security Token Service is capable of issuing SAML 1.1 and SAML 2.0
+STS is a service model defined by WS-Trust specifications. The issuance, validation, renewal, and exchange of security tokens in STS are based on WS-Trust protocol.
+
+## Security Token Service
+
+The **Security Token Service** (STS) component of WSO2 Carbon enables you to configure the generic STS to issue claim-based security tokens.
```
Comment (divyaamunugama) [inline en/docs/references/concepts/authentication/intro-ws-trust.md:8]: does general STS only issue security token?? 

validation, cancelation, renewing ???

File: en/docs/references/concepts/integrated-windows-authentication-overview.md [hunk 4]
```diff
@@ -1,116 +1,62 @@
...
 
-    -   See [Configuring IWA
-        Single-Sign-On](../../../guides/login/configure-iwa-single-sign-on) to configure
-        integrated windows authentication in WSO2 Identity Server.
--->
+However, neither Java nor the server applications that host Java web applications (like Tomcat) have native support for IWA. There are several third-party libraries that provide the ability to enable the IWA for Java web applications. Following are some of those libraries:
+
+- **JCIFS** is an open-source library that had been commonly used for IWA authentication of Java web applications. However, this library is no longer maintained and it is not recommended using due to security flaws.
+- **JESPA** is a commercial library that can be used to enable IWA in Java web applications.
+- **Tomcat IWA** is Tomcat's implementation of IWA. However, this is not fully completed yet.
```
Comment (divyaamunugama) [inline en/docs/references/concepts/integrated-windows-authentication-overview.md:57]: `**Tomcat IWA** is Tomcat's implementation of IWA. However, this is not fully completed yet.`

Is this statement  `However, this is not fully completed yet.` still valid?

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**